.github/workflows/zap_templates.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: ZAP
+
+on:
+    push:
+    pull_request:
+
+jobs:
+    templates:
+        name: ZAP templates generation
+
+        runs-on: ubuntu-latest
+
+        strategy:
+           matrix:
+            node-version: [12.x]
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+              with:
+                  submodules: true
+            - name: Use Node.js ${{ matrix.node-version }}
+              uses: actions/setup-node@v1
+              with:
+                node-version: ${{ matrix.node-version }}
+
+            - run: sudo apt-get update
+            - run: sudo apt-get install --fix-missing libpixman-1-dev libcairo-dev libsdl-pango-dev libjpeg-dev libgif-dev
```

Comment:
should we put these in the chip-build Dockerfile?

Reply:
Yes we should put them into the chip-build Dockerfile (as part of issue #3928)

Reply:
these are already in chip-build, looks like.  instead of installing here, can you run this workflow in chip-build instead?

Reply:
Some of them are missing inside chip-build (like libsdl-pango-dev ) also node fails to run in the chip-build container. As such we cannot use zap in the chip-build container until we resolve issue #3928 (which I sadly do not have time for at this moment)